socketio_block.py

```diff
@@ -230,7 +232,11 @@ def handle_data(self, data):
         data will be a dictionary, *most likely* containing an event and data
         that was sent, in the form of a python object.
         """
-        if 'event' not in data or data['event'] != 'recvData':
+        if not self.listen:
```

Comment:
This check happens pretty far down the message handling process. Is it possible to pass the flag to the client and not even do any message processing when it's received?